src/app/reactUtils.ts

```diff
@@ -166,7 +170,7 @@ export const useMatchMobileQuery = () => {
 
   React.useEffect(() => {
     matchMedia.addListener(listener);
-    return () => { matchMedia.removeListener(listener); };
+    return () => { matchMedia.removeListener(listener as any); };
```

Comment:
why is the any necessary?

Reply:
`addListener` and `removeListener` have different signatures, however, it shouldn't matter because both of these methods are deprecated and we should change them when we update `@openstax/types`

```
Argument of type '(e: MediaQueryListEvent) => void' is not assignable to parameter of type 'EventListenerOrEventListenerObject | null'.
  Type '(e: MediaQueryListEvent) => void' is not assignable to type 'EventListener'.
    Types of parameters 'e' and 'evt' are incompatible.
      Type 'Event' is missing the following properties from type 'MediaQueryListEvent': matches, mediats(2345)
```

Reply:
@TomWoodward Yes, please review, merge and deploy https://github.com/openstax/typescript-types/pull/4, then I can cherry pick changes from this PR https://github.com/openstax/rex-web/pull/1167 and fix it

Reply:
i approved and merged the one in the types repo

Reply:
You still need to deploy it right? https://www.npmjs.com/package/@openstax/types